FSubject: [PATCH] Add missing STORAGE_INTERNAL permission for BackupRestoreConfirmation

BackupRestoreConfirmation acitivity does not show 'allow' button, because
'getPasswordType' func check "Manifest.permission.STORAGE_INTERNAL",
so add missing STORAGE_INTERNAL permission.

diff --git a/packages/BackupRestoreConfirmation/AndroidManifest.xml b/packages/BackupRestoreConfirmation/AndroidManifest.xml
index 8141fa72e56..db8ebc190be 100644
--- a/packages/BackupRestoreConfirmation/AndroidManifest.xml
+++ b/packages/BackupRestoreConfirmation/AndroidManifest.xml
@@ -21,6 +21,7 @@
 
     <uses-permission android:name="android.permission.BACKUP" />
     <uses-permission android:name="android.permission.CRYPT_KEEPER" />
+    <uses-permission android:name="android.permission.STORAGE_INTERNAL"/>
 
     <application android:allowClearUserData="false"
                  android:allowBackup="false"
